src/main/java/org/testng/TestNG.java

```diff
@@ -269,28 +269,33 @@ public void setXmlPathInJar(String xmlPathInJar) {
   public void initializeSuitesAndJarFile() {
     // The Eclipse plug-in (RemoteTestNG) might have invoked this method already
     // so don't initialize suites twice.
+	System.out.println("ENTER");
```

Comment:
:) You can try to squash all your commits into only one with git interactive rebase. It's for advanced git users but it's really useful.